pkg/users/client.go

```diff
@@ -142,3 +144,62 @@ func (c *Client) GetUser(ctx context.Context, opts GetUserOpts) (User, error) {
 
 	return body, err
 }
+
+type Session struct {
+	ID        string `json:"id"`
+	Token     string `json:"token"`
+	CreatedAt string `json:"created_at"`
+	UpdatedAt string `json:"Updated_at"`
+}
+
+type AuthenticateUserWithPasswordOpts struct {
+	Email        string `json:"email"`
+	Password     string `json:"password"`
+	IPAddress    string `json:"ip_address,omitempty"`
+	UserAgent    string `json:"user_agent,omitempty"`
+	StartSession bool   `json:"start_session,omitempty"`
+	ExpiresIn    int    `json:"expires_in,omitempty"`
+}
+type AuthenticationResponse struct {
+	Session Session `json:"session"`
+	User    User    `json:"user"`
+}
```

Comment:
Should we add the comments? I'm fine with adding them later too.
I was mostly copying from the existing API reference.

Reply:
I think it's preferred, but if we're being honest in the interest of time I was skipping them for getting functionality in